kedro-hooks-tutorial/src/kedro_hooks_tutorial/hooks/model_tracking_hooks.py

```diff
@@ -56,15 +56,12 @@ def after_node_run(
         * Log the model's metrics after the model evaluating node runs.
         """
         if node._func_name == "split_data":
-            mlflow.log_params(inputs["parameters"])
+            mlflow.log_metric("parameters", inputs["params:example_test_data_ratio"])
```

Comment:
This should be `log_params({"split_data_ratio": inputs["params:example_test_data_ratio"]})`